include/maths/CBoostedTree.h

```diff
@@ -241,6 +241,9 @@ class MATHS_EXPORT CBoostedTree final : public CDataFramePredictiveModel {
     //! \warning Will return a nullptr if a trained model isn't available.
     CTreeShapFeatureImportance* shap() const override;
 
+    //! Get the selected rows that summarize \p dataFrame.
+    core::CPackedBitVector dataSummarization(const core::CDataFrame& dataFrame) const;
```

Comment:
It probably makes sense to declare this on `CDataFramePredictiveModel`. Ok we may potentially do away with this abstraction, but while it is still in the code I think we maintain it.

Reply:
added